README.md

```diff
@@ -4,20 +4,52 @@
 
-Corsica can trivially be used to control ambient displays in an office via an API. Chat bots can yield control of nearby screens to nearby desks, temporarily or indefinitely, for posting graphs, GIFs, or public announcements. Plugins can provide a regular rotation between a list of URLs for ambient dashboard display, like application metrics, promotional pages, or event calendars for the proximal space.
+Suggest content, plugin changes, and administrative changes here: https://github.com/mozilla/moz-corsica
 
-Since Corsica can push content to any modern webbrowser, clients can be monitors, tablets, or phones. Wall mounted tablets are significantly cheaper than TVs with dedicated computers. Phones can be used as transient clients to set up remote viewports when you're away from the relavent screen.
+## What is Corsica? How should I use it?
 
-Plugins make it easy to extend and add new functionality. URLs can be analyzed and reformatted to make presentation on a screen easier -- transform imgur or webcomic urls so that only the image displays, ignoring the cruft of the screen. Transform videos so that they display full screen and autoplay. Create shorthand syntaxes that unpack to whole websites:
+Corsica is a server for coordinating screens via web browsers. Content comes from an API or a plugin, is processed by one or more plugins, and sent as output to connected screens.
 
+Chat bots, browser plugins, or website widgets can use the API to control nearby screens. This is useful for Posting graphs, GIFs, slides, or public announcements. Plugins provide a playlist-like rotation from a list of URLs for ambient dashboards of application metrics, promotional pages, event calendars, etc.
+
+Since Corsica can push content to any modern webbrowser the screens can be monitors, tablets, phones, smart TVs, cars, etc.
+
+## How do I extend Corsica?
+
+Corsica aspires to a tiny core with most features implemented via plugins. Changes to the core can be submitted on this repo as PRs.
+
+Plugins are the easiest way to customize your installation and add new functionality. They are usually developed in an independent package and installed via NPM, though they can also be included directly as files.
+
+Corsica's core operates on `command`s (sometimes called `messages`), which can be submitted via the API or by plugins. Each `command` is processed by each active plugin. A plugin may choose to ignore the command, transform it, or submit a new command. `command`s are formatted like a JS function call:
+
+`command [positional1] [positional2] [kwarg=value] [kwarg2=value]`
+
+There are a few feature patterns that plugins commonly implement:
+
+1. Create an easy to remember `command` that expands into a URL, sometimes after making an external request:
+```
+xkcd random -> content type=html content='<body style="margin:0;height:100%;background:url(http://imgs.xkcd.com/comics/cloud.png) no-repeat center #000;background-size:contain;"></body>'
+```
+
+2. Reformat URLs
```

Comment:
I think these numbers won't actually work. In the rendered view, this is shown as "1.". I think that is because of the blank line, and that the code block wasn't indented. This is annoying to get right. That being said, I think you can just make this a un-ordered list instead, since it isn't really an ordered set of steps.